src/Dialogs/Settings/Panels/SymbolsConfigPanel.cpp

```diff
@@ -141,6 +142,10 @@ SymbolsConfigPanel::Prepare([[maybe_unused]] ContainerWindow &parent,
   AddBoolean(_("Fade traffic"), _("Keep showing traffic for a while after it has disappeared."),
              settings_map.fade_traffic);
 
+  AddBoolean(_("Colorful traffic"), _("show traffic in different colors depending on the climb rate and relative altitude"),
```

Comment:
I'd prefer British English - XCSoar already has a mix, but the canonical form should be British (for historical reasons; the first development team was from Australia). Also the help text should begin with upper case and end with a period.